src/liboslexec/batched_llvm_gen.cpp

```diff
@@ -8267,7 +8318,7 @@ LLVMGEN(llvm_gen_split)
             rop.llvm_load_arg(*optSep, false /*derivs*/, op_is_uniform));
     } else {
         if (op_is_uniform) {
-            args.push_back(rop.ll.constant(""));
+            args.push_back(rop.ll.constant(ustring("").c_str()));
```

Comment:
We thought this was a bug passing a char * vs. a true ustring, 
  `args.push_back(rop.ll.constant(""));`
but Turns out 
  `llvm::Value* LLVM_Util::constant(string_view s) { return constant(ustring(s)); }`
did the conversion for us already. 
Although as we move towards being more explicit, this change is probably welcome so its clear to the reader.

So 2 lines later we see a similar call:
` llvm::Value* wide_sep      = rop.ll.wide_constant("");`
 which again is handled properly by LLVM_Util, by implicitly creating a ustring.
 But perhaps we should update it to match the 2 lines before
`  llvm::Value* wide_sep      = rop.ll.wide_constant(ustring("").c_str());`
 ?

Reply:
llvm_gen_split  in 

> llvm_gen.cpp

 still uses
`   else {
        args[2] = rop.ll.constant("");
    }`
and I will change it to `rop.ll.wide_constant(ustring("").c_str());`

It might make sense to remove  
`llvm::Value* LLVM_Util::constant(string_view s) { return constant(ustring(s)); }` 
after the batched side is through with ustringhash phase3 conversion.